lib/run-in-terminal.coffee

```diff
@@ -233,6 +235,19 @@ class ArgumentsRequester
                 @save_and_hide()
                 start_terminal(@path or "", @line_edit_model.getText())
 
+switch require('os').platform()
+
+    when 'darwin'
+        default_Launchfile = """osascript -e 'tell application \"Terminal\"' -e 'activate' -e 'tell application \"Terminal\" to do script \"cd \\\"{working_directory}\\\" && {launcher} \\\"{file_path}\\\" \"' -e 'end tell'"""
+        default_Launchdir = """osascript -e 'tell application \"Terminal\"' -e 'activate' -e 'tell application \"Terminal\" to do script \"cd \\\"{working_directory}\\\" \"' -e 'end tell'"""
+
+    when 'win32'
+        default_Launchfile = 'start /D {working_directory} C:\Windows\System32\cmd.exe /u /k cd "{working_directory}" {launcher} "{file_path}"'
+        default_Launchdir = 'start /D {working_directory} C:\Windows\System32\cmd.exe /u /k cd "{working_directory}"'
+
+    else
+        default_Launchfile = 'your-favorite-terminal --foo --bar "{working_directory}" {launcher} "{file_path}"'
+        default_Launchdir = 'your-favorite-terminal --foo --bar "{working_directory}"'
```

Comment:
Ha, this should be `default_launchfile`, no capitals. AFAIK, [`snake_case`](https://en.wikipedia.org/wiki/Snake_case) is not approved for coffee, but since I'm from python-world and love coffee as «python instead of js», I prefer underscores.